.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.8","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-17.0.2.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/jamesvarghese/Library/Application Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/jamesvarghese/Library/Application%20Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
Please add `.bsp` to your `.gitignore` file - we shouldn't be committing this as it's specific to your machine

Reply:
Done